charts/fluent-bit/templates/NOTES.txt

```diff
@@ -1,3 +1,15 @@
+!WARNING! !WARNING! !WARNING! !WARNING! !WARNING!
+
+Please use the official fluent-bit chart 
+
+https://github.com/fluent/helm-charts
+
+and the official loki output
+
+https://docs.fluentbit.io/manual/pipeline/outputs/loki
+
+!WARNING! !WARNING! !WARNING! !WARNING! !WARNING!
```

Comment:
Not sure if this is the official recommendation. The plugin from Fluent may not have reached feature parity. I'd just remove that part.

Reply:
Updated the message in README and NOTES.